cli/src/ui.rs

```diff
@@ -132,7 +140,7 @@ impl Ui {
     /// Switches the output to use the pager, if allowed.
     #[instrument(skip_all)]
     pub fn request_pager(&mut self) {
-        if self.paginate == PaginationChoice::No {
+        if self.paginate == PaginationChoice::Never {
```

Comment:
```suggestion
        match self.paginate {
            PaginationChoice::Never => { return; },
            PaginationChoice::Auto => {}
        }
```

While you're here, can you change this to a `match`? That way, exhaustiveness checking will cause us to reexamine this site if/when we add an additional case.